alex/tools/kaldi/env_voip_cs.sh

```diff
@@ -39,5 +39,8 @@ export train_mmi_boost=0.05
 export mmi_beam=16.0
 export mmi_lat_beam=10.0
 
+export smbr_beam=16.0
+export smbr_lat_beam=10.0
```

Comment:
This is for dense lattice generation. It may be more important here, then
for regular decoding. Though, I did not tested properly.

Best regards,
Filip Jurcicek

-------------------------------------------------------
Work tel. (CZ): +420221914402
Personal tel. (CZ): +420777805048
Skype: bozskyfilip

http://ufal.mff.cuni.cz/filip-jurcicek
http://ufal.mff.cuni.cz/grants


On 27 February 2015 at 20:40, Ondrej Platek <notifications@github.com>
wrote:

> In alex/tools/kaldi/env_voip_cs.sh
> <https://github.com/UFAL-DSG/alex/pull/127#discussion_r25532774>:
>
> > @@ -39,5 +39,8 @@ export train_mmi_boost=0.05
> >  export mmi_beam=16.0
> >  export mmi_lat_beam=10.0
> >
> > +export smbr_beam=16.0
> > +export smbr_lat_beam=10.0
>
> NOT very important: lattice-beam value of 10 is quite large which may be
> too slow and not necessary value around 6 should be enough
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UFAL-DSG/alex/pull/127/files#r25532774>.
>